client-libs/typescript/openai-legacy/index.ts

```diff
@@ -0,0 +1,81 @@
+import * as openPipeClient from "../codegen";
+import * as openai from "openai-legacy";
+import { version } from "../package.json";
+
+// Anything we don't override we want to pass through to openai directly
+export * as openAILegacy from "openai-legacy";
+
+type OPConfigurationParameters = {
+  apiKey?: string;
+  basePath?: string;
+};
+
+export class Configuration extends openai.Configuration {
+  public qkConfig?: openPipeClient.Configuration;
+
+  constructor(config: openai.ConfigurationParameters & { opParameters?: OPConfigurationParameters }) {
```

Comment:
I think naming the arg just `openpipe` instead of `opParameters` will be clearer.

Reply:
Okay, makes sense to me!